djangoevents/tests/test_utils.py

```diff
@@ -85,6 +85,12 @@ class Evt3(DomainEvent):
             # No mutate_event present
             pass
 
+        # Marked abstract
```

Comment:
I would remove that line. You could as well write `# decorated abstract` but it's pretty much same as `@abstract` one line below